### DISCUSSION TITLE
Java版服务器招新

### DISCUSSION THREAD

**Discussion by shuiyuan2023** [share121/inter-knot#2538] [其他]
![qrcode_1723882833787](https://github.com/user-attachments/assets/51d66c37-4b4e-4f1f-b459-d5335a1153a8)

我的世界Java版1.21实验玩法服务器，支持1.8~1.20.6，支持正版和离线，手机也能玩，进群了解更多有教程

**Comment by LinkboyTvT** [comment]
有详细信息吗

**Reply by shuiyuan2023** [reply]
可以进群了解一下
